its/ruling/src/test/resources/jboss-ejb3-tutorial/java-S1258.json

```diff
@@ -1,119 +1,17 @@
 {
-'jboss-ejb3-tutorial:blob/src/org/jboss/tutorial/blob/bean/BlobEntity.java':[
```

Comment:
The missing issues are due to the fact that this project is misconfigured on purpose, since the rule now skip classes with unknown annotation, these issues dissapear.